src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -147,4 +118,17 @@ private static FileDialogConfiguration createExportFileChooser(ExporterFactory e
                 .build();
     }
 
+    private void handleError(Exception ex) {
+        LOGGER.warn("Problem exporting", ex);
+        final String errorMessage;
+        if (ex.getMessage() == null) {
+            errorMessage = ex.toString();
+        } else {
+            errorMessage = ex.getMessage();
+        }
+
+        frame.output(Localization.lang("Could not save file.") + " - " + errorMessage);
+        // Need to warn the user that saving failed!
+        frame.getDialogService().showErrorDialogAndWait(Localization.lang("Save library"), Localization.lang("Could not save file.") + "\n" + errorMessage);
```

Comment:
the `DialogService` has a method that directly accepts an exception so you don't need to compute it by hand above (and the errorMessage can be removed in the call to `output`).